clang/include/clang/Frontend/FrontendOptions.h

```diff
@@ -590,7 +596,7 @@ class FrontendOptions {
         EmitSymbolGraph(false), EmitExtensionSymbolGraphs(false),
         EmitSymbolGraphSymbolLabelsForTesting(false),
         EmitPrettySymbolGraphs(false), GenReducedBMI(false),
-        TimeTraceGranularity(500) {}
+        UseClangIRPipeline(), TimeTraceGranularity(500) {}
```

Comment:
Why isn't this initializing this to a value? I would expect it to have  'false' here.
```suggestion
        UseClangIRPipeline(false), TimeTraceGranularity(500) {}
```